### PR TITLE
Add org.profex_xrd.Profex

### PR DESCRIPTION
**Pull request by filipestevao**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
> This app uses portals by default, but `filesystem=xdg-documents` is enable because the application creates many files (useful data, conversion files, etc) automatically. This data is created in the same folder as the main open file and this a feature of Profex. I did some testing and the app crashes when no filesystem is enabled.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
> No, I'm not the upstream of this project, but the upstream was contacted by email. No link are provided because it is a not a good ideia to share email with the public. How this PR got started: I use this app in my work, and there are Windows and Mac binaries, but the Linux version needs to be compiled from source. I showed upstream that it is possible to use Flathub to share the Linux version. He liked the idea and asked me to submit and keep the flatpak version. The proof of the upstream agreement is that the appdata file has been incorporated into the source code in the new version of Profex.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> The domain of [Profex](https://www.profex-xrd.org/) belongs to Nicola Döbelin, the upstream of this project, and he is aware of and agrees to this submission.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by filipestevao** [comment]
bot, build org.profex_xrd.Profex

File: org.profex_xrd.Profex.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      - type: archive
+        url: https://www.profex-xrd.org/wp-content/uploads/2022/03/profex-5.0.1.tar.gz
+        sha256: 333d77a4b8690183bf43df1776fa4976c330f09af4799c0360bc694e47cf0c7a
+      - type: patch
+        path: bgmnbackendconfig.patch
+    post-install:
+      - mv ${FLATPAK_BUILDER_BUILDDIR}/bin /app
+      - install -Dv ${FLATPAK_BUILDER_BUILDDIR}/profex5.desktop -t /app/share/applications
+      - install -Dv ${FLATPAK_BUILDER_BUILDDIR}/profex/icons/profex5-512x512.png -t /app/share/icons/hicolor/512x512/apps
+      - mv /app/share/icons/hicolor/512x512/apps/profex5-512x512.png /app/share/icons/hicolor/512x512/apps/profex5.png
```
Comment (hfiguiere) [inline org.profex_xrd.Profex.yml:76]:
```suggestion
      - install -Dv ${FLATPAK_BUILDER_BUILDDIR}/profex/icons/profex5-512x512.png /app/share/icons/hicolor/512x512/apps/profex5.png
```

This should do the same in one line.

**Comment by filipestevao** [comment]
bot, build org.profex_xrd.Profex

**Pull request closed without merging**